### DISCUSSION TITLE
OneDrive sync issue

### DISCUSSION THREAD

**Discussion by runzhangxu** [Future-Scholars/paperlib#297] [Q&A]
Dear Developers,

Thank you for your effort in developing this wonderful Ref Manager.

I want to put the lib folder in onedrive folder and  sync it, can this cause any conflict issue in the database?

Thank you

**Comment by GeoffreyChen777** [comment]
Onedrive should be only used for syncing PDFs. The metadata database should be synced with the cloud feature. Here is the guide to  setup it: https://paperlib.app/en/doc/cloud-sync/setup.html

Please refer to https://github.com/Future-Scholars/paperlib/issues/267 for the explanation of why you cannot use Onedrive to sync the databasel.
